### ISSUE TITLE
Add support for R2019a to RTWSFCN export

### ISSUE THREAD

**Issue by t-sommer** [CATIA-Systems/FMIKit-Simulink#5]

**Comment by t-sommer** [comment]
The export on R2019b returns the following error:

```
The template makefile, X:\FMIKit-Simulink\rtwsfcnfmi\tlc\rtwsfcnfmi_18b_vcx64.tmf, must contain the |>COMPILER_COMMAND_FILE_NAME<| token if the corresponding toolchain, "MinGW64 | gmake (64-bit Windows)", has corresponding values for the RequiresCommandFile and NoCompilerCommandFile attributes.
The toolchain attributes are:

    RequiresCommandFile = true
    NoCompilerCommandFile = false

The check to determine whether the template makefile contains the token returned false.
```

@dhenriksson, can you take a look at this?

**Comment by dhenriksson** [comment]
I have analyzed this and there are simply too many changes to the build process in R2019a to have it supported at this time. I suggest that we move this to later and support only up to R2018b in version 2.6

**Comment by t-sommer** [comment]
Solved with 81dcabf94b2e81787d7ea013bb2d1d5f232b9123
